css/style.css

```diff
@@ -1,3 +1,75 @@
-h1 {
-  color: salmon;
+body {
+  height: 100vh;
+  margin: 0 auto;
+  font-family: 'Open Sans', sans-serif;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  align-items: center;
+  background-color: #ECEFEB;
+  color: #54595D;
+}
+main {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+h2 {
+  margin: 10px auto;
+  font-weight: bold;
+}
+h6 {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
Do you need these additional `font-family` declarations if you're already including one in the `body` selector?